lib/std/Uri.zig

```diff
@@ -216,6 +216,7 @@ pub fn format(
 
     const needs_absolute = comptime std.mem.indexOf(u8, fmt, "+") != null;
     const needs_path = comptime std.mem.indexOf(u8, fmt, "/") != null or fmt.len == 0;
+    const raw_path = comptime std.mem.indexOf(u8, fmt, "r") != null;
```

Comment:
The start option that triggers this is called `raw_uri`, which is arguably the more useful option. In which case this also needs to be applied to query and fragment.

Reply:
added in [710c159](https://github.com/ziglang/zig/pull/17235/commits/710c15970ce9bfd31074ecfb3fd63147f077b87a)